GEOSdatasea_GridComp/DataSea_ExtData.yaml

```diff
@@ -0,0 +1,33 @@
+Exports:
+    DATA_UW:
+        collection: /dev/null
+    DATA_VW:
+        collection: /dev/null
+    DATA_DW:
+        collection: /dev/null
+        linear_transformation: [-3682.0, 0.0]
+
+# --
+#
+# Comment:
+#    configuration: GEOS/WGCM
+#    description: realistic currents and ocean depth
+#
+# Samplings:
+#    static:
+#    extrapolation: persist_closest
+#
+# Collections:
+#     gebco_terrain_model_for_ocean_and_land:
```

Comment:
These must be defaulted to _current_ system(s), instead of S2Sv2. 

Specifically for DW please use use same topography as real ocean- yes, even in data mode, it must be realistic!

Reply:
@sanAkel - I do not follow. I am not sure why S2S was mentioned in your comment. I've set the defaults to replicate the hardcoded values (currents=0) in the datasea GC and get zero-diff. IMO, the const value of DATA_DW is a reasonable compromise as it avoids unnecessary I/O, because as of the time of this PR, only WGCM needs DW. I could have made the DW export conditional on WGCM being active, but this would break symmetry with the other datasea exports and IMO is not needed.

Reply:
@adarmenov 

[This line](https://github.com/GEOS-ESM/GEOS_OceanGridComp/blob/d6e0b9fde40a442f99fc8f142e2176e52b95d717/GEOSdatasea_GridComp/DataSea_ExtData.yaml#L23-L24) is reason why I said: ``These must be defaulted to current system(s), instead of S2Sv2."

> I do not follow. I am not sure why S2S was mentioned in your comment.

Hope it makes sense.


In any case, as you say, only `DW` suffices 

> I've set the defaults to replicate the hardcoded values (currents=0) in the datasea GC and get zero-diff. IMO, the const value of DATA_DW is a reasonable compromise as it avoids unnecessary I/O, because as of the time of this PR, only WGCM needs DW.

Please take off that reference to some (s2s output file]((https://github.com/GEOS-ESM/GEOS_OceanGridComp/blob/d6e0b9fde40a442f99fc8f142e2176e52b95d717/GEOSdatasea_GridComp/DataSea_ExtData.yaml#L23-L24) so we aren't confused later.

Also, as for `DW` please use the topography I pointed earlier. Thanks.

Reply:
@sanAkel, I've restructured the code to narrow down the scope of depth to WGCM. I expect that the coupled O-A configuration will work out of the box with these changes.